applications/examples/example_event_loop/application.fam

```diff
@@ -0,0 +1,32 @@
+App(
+    appid="example_event_loop_timer",
+    name="Example: Event Loop Timer",
+    apptype=FlipperAppType.EXTERNAL,
+    entry_point="example_event_loop_timer_app",
+    fap_category="Examples",
+)
```

Comment:
better specify sources=[] for each app to avoid compiling all files multiple times